inception-pdf-editor/src/test/java/de/tudarmstadt/ukpinception/pdfeditor/pdfanno/model/PdfAnnoRendererTest.java

```diff
@@ -171,7 +171,7 @@ public void testRender() throws Exception
 
         PdfExtractFile pdfExtractFile = new PdfExtractFile(pdftxt, new HashMap<>());
         PdfAnnoModel annoFile = PdfAnnoRenderer.render(state, vdoc,
-            cas.getDocumentText(), schemaService, pdfExtractFile);
+            cas.getDocumentText(), schemaService, pdfExtractFile, Offset.empty());
```

Comment:
If you pass `Offset.empty()` here, will the annoFile contain nothing?

Reply:
No, this offset basically is the page offset.
E.g. if we render page 5 of a document this would contain the character offset to the beginning of the 5th page.
As in the test case we render the whole document we don't need an offset.
I probably should add tests for this with offset other than 0

Reply:
If 0 is a valid offset, then maybe `Offset.empty()` should be `[-1,-1]` ? Or what is the purpose of this empty offset? I assume it is used to indicate a default invalid value.

Reply:
Some JavaDoc would be good here ;)

Reply:
No it is fine as it is, it just indicates we do not have some offset for rendering here.
The offset begin value is added to each span.
If we would have like page 2, the offset would be the index of page 2 begin.
Here it is just at the begin of the document, hence offset 0.
Maybe `empty` is a misleading name.
But as you stated somewhere else in the review we don't need an offset here, just the begin of the offset is required.
I will make changes according to the review now and will add a short comment on this test.